### PR TITLE
Dream Texture: UI Changes

### PR DESCRIPTION
**Pull request by blastframe**
image

- Changed UI from what was a tedious three-click process to generate a new Dream Texture image to a one-click process.
- "Dream Texture" text was very wide in the Image Editor & Node Editor menus & hard to access, now in collapsible sidebar
- Organized UI module into folder
- Assigned more appropriate History icon
- Smaller UI tweaks

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 bl_info = {
     "name": "Dream Textures",
-    "author": "Carson Katri",
+    "author": "Carson Katri, Kevin C. Burke",
```
Comment (carson-katri) [inline __init__.py:16]: If this should include all contributors, can you include Greg Richardson (@gregnr)?

```suggestion
    "author": "Carson Katri, Greg Richardson, Kevin C. Burke",
```

File: ui/panel.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        advanced_box.prop(scene.dream_textures_prompt, "cfgscale")
+        advanced_box.prop(scene.dream_textures_prompt, "sampler")
+        advanced_box.prop(scene.dream_textures_prompt, "show_steps")
+    
+    row = layout.row()
+    row.operator(ViewHistory.bl_idname, icon="RECOVER_LAST")
+    row.operator(HelpPanel.bl_idname, icon="QUESTION", text="")
+    row = layout.row()
+    row.scale_y = 1.5
+    row.operator("shade.dream_texture", icon="PLAY", text="GO")
```
Comment (carson-katri) [inline ui/panel.py:74]:
```suggestion
    row.operator("shade.dream_texture", icon="PLAY", text="Generate")
```

**Review by carson-katri — CHANGES_REQUESTED** [review]
This looks really nice, thanks for the contribution! I had a few suggestions, let me know what you think.

**Comment by blastframe** [comment]
Thank you @carson-katri for the feedback and incredible add-on! I've accepted the changes you requested.

**Comment by blastframe** [comment]
Oops, I accidentally closed it. It should be ready to merge!

**Comment by carson-katri** [comment]
Thanks for the extra fixes!

**Pull request merged**